### ISSUE TITLE
ggt dev is crashing

### ISSUE THREAD

**Issue by callummcmaugh-er** [gadget-inc/ggt#1330]
### Error ID

afdf8a94-04a1-4487-8bcd-0d9a80ae98c2

### What happened?

I'm running ggt dev command for my project.

The sync stays active for about 30 seconds before crashing.

```
An error occurred while communicating with Gadget
Unexpected server response: 404
```

I'm having to rerun the command everytime I want to push a change. This is severly slowing down the development process..

### What did you expect to happen?

Sync to stay active, like previous GGT versions

### Anything else?

ggt version 1.01

Extra error IDS:
afdf8a94-04a1-4487-8bcd-0d9a80ae98c2
308f13c5-5753-411d-ac0a-9971ae3427f4
1af0aa23-38e5-44e7-872a-7310232401d0
6ecbf010-a65e-4497-9ab2-dd6168015c31

**Comment by scott-rc** [comment]
Sorry about this! I'll look into this today.

**Comment by scott-rc** [comment]
Hey @callummcmaugh-er,

I just released [v1.0.2](https://github.com/gadget-inc/ggt/releases/tag/v1.0.2) which should fix this issue. Please re-open this issue if the `Unexpected server response: 404` error continues on the latest version!
